book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] As mentioned by the previous reviewer you should ensure that `rentals`  has a getter and setter which means the `rentals` should be `attr_accessor`, please kindly fix that 🙃 😁 
- [ ] Also is the case for the `person.rb` file, the `rentals` should have a getter and setter 😉 😉

Reply:
Thanks @cynthiainga